### DISCUSSION TITLE
Sizing Estimation

### DISCUSSION THREAD

**Discussion by mbarnettHMX** [microsoft/PubSec-Info-Assistant#389] [Q&A]
In the [Sample Azure Estimation](https://azure.com/e/06376dd6bb454d088b7af5fdd6e00d48) example, the monthly cost shown is substantially due to the cost of the Azure AI Document Intelligence (AIDI) service. Since many customers will have a large corpus of documents (for example, 1TB or ~60M pages) this amounts to $27.2K for AIDI. Can we assume this is a one-month charge, or would such a large dataset impose other resource sizing requirements?

![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/123389082/c0ce977b-5309-41ce-b4d0-a997864a5dae)

**Comment by lon-tierney** [comment]
The Document Intelligence cost is generally going to be a one-time cost, but there will be the associated ongoing storage costs, as well as search index, so I would size in your documents/storage costs accordingly.
